### PR TITLE
Use workon -n or --no-cd 

### PR DESCRIPTION
**Pull request by ashwinvis**
* Changed `--here` option to `--no-cd` to be similar to https://virtualenvwrapper.readthedocs.io/en/latest/command_ref.html#workon

### CODE REVIEW

**Comment by pfmoore** [comment]
Changing to use argparse seems good to me - it means that `pew workon --help` would work. But I'm not sure if changing `--here` to --no-cd` is OK - this will break for people who currently use `--here`. Obviously, it's not a huge fix for users to change the option name, but is the benefit (partial compatibility with virtualenvwrapper, we still don't support `--cd`) worth it?

Maybe this PR could be split in two - the change to argparse (which is fine, IMO) and the argument name change (which may need more discussion)?

@ashwinvis Apologies for the delay in reviewing this PR - are you still interested in working on this?

**Comment by ashwinvis** [comment]
>  this will break for people who currently use--here

It is not too late to change it as the `--here` option is unreleased.

> Maybe this PR could be split in two - the change to argparse (which is fine, IMO) and the argument name change (which may need more discussion)?

Let me see if I can split the PR.

**Comment by ashwinvis** [comment]
Another motivation is `--here` cannot be shortened as `-h` as it is already taken up by `--help`. One would have to go for `-e` or `-r` which is not so intuitive I think.

**Comment by pfmoore** [comment]
> It is not too late to change it as the --here option is unreleased.

Sorry, I've been using a version of pew from master for so long now (because the released version had issues with Python 3.7) that I'd forgotten this fact. Given this, I agree that changing the option name now rather than later makes sense.

**Comment by pfmoore** [comment]
I've approved #205 and I'll merge it in a while unless @berdario has any concerns with it. Once that is merged, this PR should become a lot cleaner (at the moment it seems to include the argparse stuff as well as the option name change) and I'll review it then. The only thing I'd say right now is that in the readme, `usage: pew workon [-h] [-n] [envname]`, I'd rather we used the long form of the option, as it's clearer.

File: README.md [hunk 1]
```diff
@@ -170,9 +170,9 @@ The `-r` option can be used to specify a text file listing packages to be instal
 
 List or change working virtual environments.
 
-`usage: pew workon [environment_name] [--here]`
+`usage: pew workon [-h] [-n] [envname]`
```
Comment (berdario) [inline README.md]: As pfmoore mentioned, can you please use the long form of the option?

Otherwise, it looks good

**Comment by ashwinvis** [comment]
@berdario Thanks for merging and feel free to change it to something else if you like. I only want a one letter shorthand for this option. Typing `--here` everytime is painful.

**Pull request merged**